packages/tools/devtools/devtools-view/src/test/SettingsView.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import { debug } from "jest-preview";
+import React from "react";
+
+import { SettingsView } from "../components/index.js";
+
+import { assertNoAccessibilityViolations } from "./utils/index.js";
+
+describe("SettingsView Accessibility Check", () => {
+	it("SettingsView is accessible", async () => {
+		const { container } = render(<SettingsView />);
+		await assertNoAccessibilityViolations(container);
+	});
+
+	it("Can tab/arrow navigate through the SettingsView", async () => {
+		render(<SettingsView />);
+
+		const user = userEvent.setup();
+		// Focus on the first interactive element (dropdown theme selector)
+		await user.tab();
+		const dropdown = screen.getByRole("combobox", { name: /Theme Selection Dropdown/ });
```

Comment:
Want to verify if `combobox` is the right option. 

`SettingsView`'s theme dropdown menu uses `Dropdown` component. MDN [documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role) suggests `ARIA: combobox role` is an element as an input that controls another element, such as a `listbox` or `grid`. 

Can we consider a dropdown menu as subset of one of the two items?

Reply:
Ignore the comment 😄.

_"The combobox role is used to indicate that the element is a composite widget made up of a single-line textbox and an associated listbox, which may be displayed by clicking on or otherwise interacting with the textbox."_

I think `combobox` is the right choice.

Reply:
Yes, combobox is the correct option. It's being used to control/select from multiple theme options. You can verify this by changing the role. Running the test will produce a failure saying an element with that role  wasn't found and the accessible role is `combobox:` with the name `Theme Selection Dropdown.`